backend/src/test/relay/common.ts

```diff
@@ -48,7 +48,7 @@ export const setup = async () => {
   await cryptoWaitReady();
 
   const signer = new PoolSigner(
-    registry,
+    registry as unknown as Registry,
```

Comment:
Why is this necessary?

Reply:
yes, due to changes in the Polkadot.js API

Reply:
Hm, what exactly has changed there? I would imagine to registry be just registry.

Reply:
After update: `Argument of type 'TypeRegistry' is not assignable to parameter of type 'Registry'`. This did not affect any application code, only this line in tests

Reply:
Maybe you need to do `new Registry` instead of `new TypeRegistry` now?

Reply:
No, I tried that. I can investigate that